at16k/core/media.py

```diff
@@ -18,15 +18,15 @@ def sample_rate(self):
         """
         Sample rate of audio file
         """
-        sample_rate, _ = wavfile.read(self.file_path)
+        sample_rate = librosa.get_samplerate(path=self.file_path)
         return sample_rate
 
     @property
     def waveform(self):
         """
         Raw waveform of entire audio file
         """
-        _, waveform = wavfile.read(self.file_path)
+        waveform, _ = librosa.load(path=self.file_path, sr=None)
         shape = np.shape(waveform)
         if len(shape) > 1:
             waveform = np.mean(waveform, axis=1)
```

Comment:
Just looked up librosa docs, axis=0 would be the correct way to average across channels. Can you make the change and update the PR?